route/v1/types.go

```diff
@@ -70,7 +70,23 @@ type RouteSpec struct {
 	// chosen.
 	// Must follow DNS952 subdomain conventions.
 	Host string `json:"host" protobuf:"bytes,1,opt,name=host"`
-	// Path that the router watches for, to route traffic for to the service. Optional
+	// subdomain is a DNS subdomain that is requested within the ingress controller's
+	// domain (as a subdomain). If host is set this field is ignored. An ingress
+	// controller may choose to ignore this suggested name, in which case the controller
+	// will report the assigned name in the status.ingress array or refuse to admit the
```

Comment:
Looking at this and the existing docs as a whole, is it clear that regardless of whether Host or Subdomain is in use, the admitted/effective host name is reflected in `.status.ingress`?

Reply:
Tes